tools/diff-processor/rules/rules_field.go

```diff
@@ -80,12 +80,12 @@ var fieldRule_BecomingRequired = FieldRule{
 }
 
 func fieldRule_BecomingRequired_func(old, new *schema.Schema, mc MessageContext) string {
-	// Ignore for removed fields
-	if new == nil {
+	// Ignore for added / removed fields
```

Comment:
Do we want to add a ticket for this

 **adding a required field**

Reply:
added b/300515447